proofs/pfedit.ml

```diff
@@ -233,10 +233,10 @@ let declare_implicit_tactic tac = implicit_tactic := Some tac
 
 let clear_implicit_tactic () = implicit_tactic := None
 
-let solve_by_implicit_tactic env sigma evk =
+let apply_implicit_tactic tac = (); fun env sigma evk ->
```

Comment:
My Ocaml-foo is rather lacking. Could you explain to me why it is better to have a fully-applied one-argument function that returns a closure rather than to have a partly-applied four-argument closure? I know that partly-applied closures had poor performances in the past, but I thought this Ocaml bug had been fixed years ago.